src/createUseQuery.mts

```diff
@@ -238,18 +244,29 @@ export function createQueryHook({
   requestParams,
   method,
   className,
+  pageParam,
+  nextPageParam,
 }: {
-  queryString: "useSuspenseQuery" | "useQuery";
+  queryString: "useSuspenseQuery" | "useQuery" | "useInfiniteQuery";
   suffix: string;
   responseDataType: ts.TypeParameterDeclaration;
   requestParams: ts.ParameterDeclaration[];
   method: MethodDeclaration;
   className: string;
+  pageParam?: string;
+  nextPageParam?: string;
 }) {
   const methodName = getNameFromMethod(method);
   const customHookName = hookNameFromMethod({ method, className });
   const queryKey = createQueryKeyFromMethod({ method, className });
 
+  const isInfiniteQuery =
```

Comment:
should we throw an exception if 'queryString' is not set to 'useInfiniteQuery' and either pageParam or nextPageParam is undefined?